test/dotnet-new3.UnitTests/Approvals/AllWebProjectsWork.CanShowHelp_Mvc.approved.txt

```diff
@@ -4,7 +4,7 @@ Description: A project template for creating an ASP.NET Core application with ex
 This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/5.0-third-party-notices for details.
 
 Usage:
-  new3 mvc [options] [template options]
+  dotnet-new3 new3 mvc [options] [template options]
```

Comment:
the addition of `dotnet-new3` here is because there needed to be a rootcommand to host the `new3` command, right? so in the SDK we'd see `dotnet` here instead?